### PR TITLE
add Japanese translation for the Headless Chrome post

### PR DESCRIPTION
**Pull request by myakura**
which is https://developers.google.com/web/updates/2017/04/headless-chrome

### CODE REVIEW

**Comment by myakura** [comment]
@agektmr can you take a look at it?

File: src/content/ja/updates/2017/04/headless-chrome.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: ヘッドレス Chrome とはなにか、そしてその使い方を紹介します。
+
+{# wf_updated_on: 2017-04-27 #}
+{# wf_published_on: 2017-04-27 #}
```
Comment (jpmedley) [inline src/content/ja/updates/2017/04/headless-chrome.md]: We're missing some metadata that's [in the original](https://raw.githubusercontent.com/google/WebFundamentals/master/src/content/en/updates/2017/04/headless-chrome.md). They're required for all translations.
* wf_tags
* wf_featured_image
* wf_featured_snippet

File: src/content/ja/updates/2017/04/headless-chrome.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+<style>
+figure {
+  text-align: center;
+}
+</style>
+
+### TL;DR {: #tldr .hide-from-toc}
+
+[Headless Chrome](https://chromium.googlesource.com/chromium/src/+/master/headless/README.md) は Chrome をヘッドレス環境で実行する手段です。Chrome をクローム（ブラウザーのUIのこと）なしに実行します！ヘッドレス Chrome によって、Chromium とそのエンジン Blink が提供する**モダンなウェブプラットフォームの機能すべて**がコマンドラインにもたらされるのです。
+
```
Comment (jpmedley) [inline src/content/ja/updates/2017/04/headless-chrome.md]: Please break lines at 80 characters.

**Comment by kosamari** [comment]
Hey @myakura thanks for the translation ! 
Heads up, more edit to the post was just merged & will be deployed shortly.  There are 2 large paragraph added you might want to include (diff: https://github.com/google/WebFundamentals/commit/11fee6177962db2d26ca43b52bbb73110f3dadc7) 
thanks !

File: src/content/ja/updates/2017/04/headless-chrome.md (hunk 2 above)
Comment (myakura) [inline src/content/ja/updates/2017/04/headless-chrome.md]: doing so in Japanese isn't great as line breaks create whitespaces and that's awkward to the Japanese.

File: src/content/ja/updates/2017/04/headless-chrome.md (hunk 1 above)
Comment (myakura) [inline src/content/ja/updates/2017/04/headless-chrome.md]: oh, I didn't see any other Japanese translations include them so I took out. okay. I'll re-add them.

**Comment by myakura** [comment]
@kosamari thanks. totally forgotten to fetch upstream :-(

File: src/content/ja/updates/2017/04/headless-chrome.md (hunk 1 above)
Comment (myakura) [inline src/content/ja/updates/2017/04/headless-chrome.md]: addressed in 29d7680.

**Pull request merged**